Source/JavaScriptCore/assembler/ARM64Assembler.h

```diff
@@ -514,8 +514,8 @@ class ARM64Assembler {
     enum Datasize {
         Datasize_32,
         Datasize_64,
-        Datasize_64_top,
-        Datasize_16
+        Datasize_128,
+        Datasize_16,
```

Comment:
Can this `Datasize_16` be put at the top for proper sorting?  Does the ordering here actually matter?  If so, this deserves a comment at minimum.